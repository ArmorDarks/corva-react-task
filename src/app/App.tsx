/* eslint no-unused-vars: "off" */
/* eslint @typescript-eslint/no-unused-vars: "error" */

import { hot } from 'react-hot-loader/root'
import React from 'react'

import Data from './Data'
import './styles/index.scss'

const App = () => (
  <div>
    <h1>Hello, world!</h1>
    <Data />
  </div>
)

export default hot(App)
