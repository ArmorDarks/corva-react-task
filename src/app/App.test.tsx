/* eslint-env jest */
/* eslint no-unused-vars: "off" */
/* eslint @typescript-eslint/no-unused-vars: "error" */

import React from 'react'
import { shallow } from 'enzyme'

import App from './App'

describe('App', () => {
  it('should render', () => {
    expect(shallow(<App />)).toMatchSnapshot()
  })
})
