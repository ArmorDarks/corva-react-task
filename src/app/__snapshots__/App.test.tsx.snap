// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
<h1>
  Hello, world!
</h1>
`;
